bundles/org.openhab.core.addon/src/test/java/org/openhab/core/addon/AddonInfoRegistryMergeTest.java

```diff
@@ -67,7 +60,7 @@ private AddonInfoProvider createAddonInfoProvider0() {
 
     private AddonInfoProvider createAddonInfoProvider1() {
         AddonDiscoveryMethod discoveryMethod = new AddonDiscoveryMethod().setServiceType("mdns")
-                .setMdnsServiceType("_hue._tcp.local.");
+                .setParameters(List.of(new AddonParameter("mdnsServiceType", "_hue._tcp.local.")));
```

Comment:
"mdnsServiceType" should be a constant

Reply:
Left as is, as this would also require a local constant and the exact name of the parameter is not relevant for the test.